packages/client/src/stores/notification.js

```diff
@@ -18,25 +16,27 @@ const createNotificationStore = () => {
     type = "info",
     icon,
     autoDismiss = true,
+    duration = 3000,
```

Comment:
Defaulting this here accounts for `undefined` only, but not for `null` or the more common `""` (empty string) which will happen if users touch the new setting but don't enter anything. So I'd prefer to remove the default here and instead apply it below.